src/app-layout/__integ__/toolbar-tooltips/app-layout-toolbar-split-panel-trigger-tooltip.test.ts

```diff
@@ -12,10 +14,21 @@ describe('refresh-toolbar', () => {
     //matches drawerItems[0].id from '../../../../lib/dev-pages/pages/app-layout/utils/drawers';
     const firstDrawerId = 'security';
     const firstDrawerTriggerSelector = wrapper.findDrawerTriggerById(firstDrawerId).toSelector();
-    const splitPanelTriggerSelector = wrapper.findSplitPanelOpenButton().toSelector();
+    const splitPanelTriggerSelector = wrapper.findByClassName(testutilStyles['split-panel-trigger']).toSelector();
     const tooltipSelector = wrapper.findDrawerTriggerTooltip().toSelector();
     const expectedTooltipText = 'Open panel';
 
+    async function focusSplitPanelTriggerWithKeyboard(page: AppLayoutDrawersPage) {
+      await expect(page.isExisting(tooltipSelector)).resolves.toBe(false);
+      // Click on the last breadcrumb (which is not a link)
+      await page.click(wrapper.findBreadcrumbs().findBreadcrumbGroup().findBreadcrumbLink(2).toSelector());
+      await page.keys(['Tab', 'Tab']); // Focus the first drawer trigger past the split panel trigger
+      await page.keys(['Shift', 'Tab']); // Tab back to the split panel trigger
```

Comment:
Do we need to do this back-and-forward focus in every test? Is it really worth reusing this code? For example, you could only reuse the assertion part.